### PR TITLE
FIX: pass BAM files to metaquast in the correct order

### PR DESCRIPTION
**Pull request by misialq**
The BAM file paths passed to the metaQUAST command (if provided) have the wrong order - they need to be sorted so that the order matches the order of the contig files.

### CODE REVIEW

**Comment by DorielaGrabocka** [comment]
Tests pass, the command passes the BAM files in the same order as contigs.

File: q2_assembly/quast/quast.py [hunk 1]
```diff
@@ -338,10 +334,13 @@
     Returns:
         a Pandas dataframe with the tabular data.
     """
-    filename = "transposed_report.tsv"
-    filepath = os.path.join(results_dir, "combined_reference", filename)
+    subdir = os.path.join(results_dir, "combined_reference")
+    if os.path.isdir(subdir):
```
Comment (DorielaGrabocka) [inline q2_assembly/quast/quast.py:338]: Do we do this to keep the results in the combined reference folder returned by QUAST?

**Review by DorielaGrabocka — APPROVED** [review]

**Comment by DorielaGrabocka** [comment]
Only issue, coverage below threshold.

File: q2_assembly/quast/quast.py (hunk 1 above)
Comment (misialq) [reply q2_assembly/quast/quast.py:338]: I think it happened to me that this folder did not exist (when references were not fetched), in which case the transposed_report was written somewhere else and something failed after this step so I added this little check :)

**Pull request merged**